README.md

```diff
@@ -82,17 +82,55 @@ To use MoveIt some additional packages should be added into workspace:
+There are many possible ways to connect a UR robot. This section describes a good example using static IP adresses and a direct connection from the PC to the Robot to minimize latency introduced by network hardware. Though a good network switch usually works fine, as well.
+
+1. Connect the UR control box directly to the remote PC with an ethernet cable.
+
+2. Open the network settings from the UR teach pendant (Setup Robot -> Network) and enter these settings:
+
+```
+IP address: 192.168.1.102
+Subnet mask: 255.255.255.0
+Default gateway: 192.168.1.1
+Preferred DNS server: 192.168.1.1
+Alternative DNS server: 0.0.0.0
+```
+
+3. On the remote PC, turn off all network devices except the "wired connection", e.g. turn off wifi.
+
+4. Open Network Settings and create a new Wired connection with these settings. You may want to name this new connection `UR` or something similar:
+
+```
+IPv4
+Manual
+Address: 192.168.1.101
+Netmask: 255.255.255.0
+Gateway: 192.168.1.1
+```
+
+5. Verify the connection from the PC with e.g. ping.
+
+```
+ping 192.168.1.102
+```
+
+## Connect to External Control via URCap
+
+This section describes installation and launching of the URCap program from the pendant. It allows ROS to control the robot externally. Generally, you will launch the driver via ROS then start URCap from the pendant.
+
+1. To enable external control of the UR robot from a remote PC you need to install the **externalcontrol-1.0.5.urcap** which can be downloaded from [Universal_Robots_ExternalControl_URCap](https://github.com/UniversalRobots/Universal_Robots_ExternalControl_URCap/releases).
 
 **Note:** For installing this URCap a minimal PolyScope version of 3.7 or 5.1 (in case of e-Series) is necessary.
 
 2. For installing the necessary URCap and creating a program, please see the individual tutorial on how to [setup a CB3 robot](/ur_robot_driver/doc/install_urcap_cb3.md) or how to [setup an e-Series robot](/ur_robot_driver/doc/install_urcap_e_series.md)
 
 3. On the remote PC, launch the suitable launch file which starts the robot driver and controllers (see details in [Usage](#usage) section).
 
-4. In the Program tab of the teach pendant, navigate to the URCaps section on the left and add the external control to the robot program by clicking on it. The program can then be executed by pressing the play button. Make sure the robot is turned on. The robot power status will be displayed on the bottom left.
+4. From the Program Robot tab of the teach pendant, load `external_control.urp`. Click on the "Control by..." section of the program to check the Host IP of the external PC. If it needs to be modified, make the modification under the Installation tab (as prompted on screen). You do not need to modify the Custom Port.
```

Comment:
Please verify that the same external_control.urp works  on both CB3 and e-Series. It should but please verify be we get a lot of issues

Reply:
@MGBla can you confirm that the teach pendant script you run on an e-Series is called `external_control`? Bonus if you can confirm it's v1.0.5 ([from here](https://github.com/UniversalRobots/Universal_Robots_ExternalControl_URCap/releases))

I can confirm it works for CB3 robots

Reply:
OK, I got confirmation from a PickNik employee that they use the same script on a UR5e.